libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-status.component.ts

```diff
@@ -47,5 +47,6 @@ import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
   `,
 })
 export class DhMeteringPointStatusComponent {
-  status = input.required<'CLOSED' | 'CONNECTED' | 'DISCONNECTED'>();
+  // 'CLOSED' | 'CONNECTED' | 'DISCONNECTED'
+  status = input.required<string>();
```

Comment:
suggestion: looks like "UNKNOWN" status is also sent in. Let's handle that too.

Reply:
This will become a enum at some point, will do it there :-)